cscs-checks/prgenv/ssh_environ_check.py

```diff
@@ -0,0 +1,18 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class SSHLoginEnvCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.descr = 'Test the environment when accessed remotely over SSH'
+        self.valid_systems = ['daint:login', 'dom:login']
+        self.valid_prog_environs = ['PrgEnv-cray']
+        self.executable = "ssh"
+        self.executable_opts = ["%s" % self.current_system.name,
+                                "echo", "'$SCRATCH'"]
+        self.sanity_patterns = sn.assert_found(
+            r'/scratch/[\S+]', self.stdout)
+        self.maintainers = ['RS', 'VK']
+        self.tags = {'maintenance'}
```

Comment:
Add also production so that we can catch if there is something problematic with the profiles.